### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by istankovic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+* deployments: updated mainnet deployment
+* contracts: major simplifications, fixes and optimizations
+* contracts: audit-related fixes
+* contracts: add support for Arbitrum
+* contracts: allow withdrawals on behalf of others
+* contracts: allow claims on behalf of others
+* contracts: allow claims for a limited time after request expiry
+* contracts: set minValidityPeriod to 30 minutes
+* contracts: set maxValidityPeriod to 48 hours
+* agent: add unsafe-fill-time option
```
Comment (GabrielBuragev) [inline CHANGELOG.md:14]: Please mention the multi-token support

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## v1.0.0 (2023-01-12)
+
+* deployments: updated mainnet deployment
+* contracts: major simplifications, fixes and optimizations
```
Comment (fredo) [inline CHANGELOG.md:6]: I think it's worth to mention the merge of the resolution registry into the request manager explicitly

**Review by fredo — APPROVED** [review]
looks good from my side

File: CHANGELOG.md (hunk 2 above)
Comment (istankovic) [reply CHANGELOG.md:6]: Done.

**Pull request merged**